packages/react-components/react-menu/src/components/MenuItemLink/__snapshots__/MenuItemLink.test.tsx.snap

```diff
@@ -6,6 +6,7 @@ exports[`MenuItemLink renders a default state 1`] = `
     class="fui-MenuItemLink fui-MenuItem"
     href="#"
     role="menuitem"
+    tabindex="0"
```

Comment:
Seems like the `baseState` introduces some extra unnecessary tabIndex=0, we can easily remove it: https://github.com/microsoft/fluentui/pull/30497/files/a7ccbce6deae3edae71f5cd3fe71134c28dc8eb2..b79067e0bcceb977204c59bb03a48166d8c3f614#r1486318397

Reply:
Changed!